docs/sphinx/examples/python/trotter_kernel_mode.py

```diff
@@ -33,7 +33,7 @@
 # This is because the CPU-only backend has difficulty handling
 # 30+ qubit simulations.
 
-spins = 5  # set to around 25 qubits for `nvidia` target
+spins = 11  # set to around 25 qubits for `nvidia` target
```

Comment:
The C++ example says to set spins = 100 and steps = 25 for nvidia target, but they are reversed here. Which one is the better recommendation?

Reply:
looks like a typo in c++ one, thanks!

Reply:
fixed!